src/modules/client_service.js

```diff
@@ -423,26 +426,27 @@ export const clientService = {
     /**
      * {string} Constant Semantic versioning, minimum client (BPL/BPC) allowed
      */
-    MINIMUM_ALLOWED: '0.7.0',
+    MINIMUM_ALLOWED: '1.0.1',
 
     /**
      * {string} Constant Semantic versioning, minimum recommended
      * client/launcher version
      */
     RECOMMENDED: '1.0.1',
 
-    /**
-     * {string} Constant Semantic versioning, Minimum client/launcher version
-     * supporting coded/verbose responses.
-     * NOTE: (remove after MINIMUM_ALLOWED > this)
-     */
-    CODED_MINIMUM: '0.7.5',
+    // /**
```

Comment:
Remove these comments. They are no longer required.

Reply:
Completed in the next push.